packages/core/primitives/signals/src/computed.ts

```diff
@@ -6,10 +6,11 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
+import {RuntimeError, RuntimeErrorCode} from '../../../src/errors';
```

Comment:
This shouldn't be a deep import ! 
```suggestion
import {ɵRuntimeErrorCode as RuntimeError, ɵRuntimeErrorCode as RuntimeErrorCode} from '../../../src/errors';
```

Reply:
What kind of import should it be instead? What do you mean by deep? @JeanMeche